app/(dashboard)/finances/page.tsx

```diff
@@ -11,8 +14,20 @@ import EmbeddedComponentContainer from '@/app/components/EmbeddedComponentContai
 import {LandmarkIcon, LoaderCircle} from 'lucide-react';
 import {Button} from '@/components/ui/button';
 import {useFinancialAccount} from '@/app/hooks/useFinancialAccount';
+import { FinancingProductType } from '@stripe/connect-js';
 
 export default function Finances() {
+
+  // Only show the financing offer if there is one to show
+  const [showFinancingOffer, setShowFinancingOffer] = React.useState(false);
+  const handleFinancingOfferLoaded = ({ productType }: FinancingProductType) => {
+    if (productType && productType != 'none') {
+      setShowFinancingOffer(true);
+    } else {
+      setShowFinancingOffer(false);
+    }
```

Comment:
slight nit: I'd use a switch statement here to exhaustively handle all `productType`s. In addition, `productType` is never `undefined`, so we can limit it to just checking for values in the enum too.

```suggestion
    switch (productType) {
      case 'none':
        setShowFinancingOffer(false);
        break;
      case 'standard':
      case 'refill':
        setShowFinancingOffer(true);
        break;
    }
```